src/main/java/me/devsnox/playtime/commands/PlayTimeCommand.kt

```diff
@@ -0,0 +1,87 @@
+package me.devsnox.playtime.commands
+
+import me.devsnox.playtime.playtime.TimeManager
+import me.devsnox.playtime.utils.Messages
+import org.bukkit.Bukkit
+import org.bukkit.command.Command
+import org.bukkit.command.CommandExecutor
+import org.bukkit.command.CommandSender
+import org.bukkit.entity.Player
+import java.util.concurrent.TimeUnit
+
+/**
+ * Created by Yasin Dalal
+ * Copyright (c) 2017-2019 Yasin Dalal
+ * GitHub: https://github.com/DevSnox
+ * E-Mail: yasin@dalal.ch
+ */
+class PlayTimeCommand(private val timeManager: TimeManager) : CommandExecutor {
+    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
```

Comment:
change the return type to void. I think the best way ist to create a another function